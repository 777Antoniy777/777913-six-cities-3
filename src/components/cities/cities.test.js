import React from 'react';
import renderer from "react-test-renderer";
import Cities from './cities';

// set mocha data
const cities = [
  `Omsk`,
  `Moscow`,
  `Saint-Petersburg`,
];
const currentCity = `city`;

const onGetCity = () => {};

it(`render Cities`, () => {
  const tree = renderer.create(
      <Cities
        cities={cities}
        currentCity={currentCity}
        onGetCity={onGetCity}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
