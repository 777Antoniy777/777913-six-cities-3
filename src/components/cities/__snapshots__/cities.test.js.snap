// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Cities 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Omsk
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Moscow
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        Saint-Petersburg
      </span>
    </a>
  </li>
</ul>
`;
